rules/rules-overridden-azure/cloud-readiness/socket-communication.windup.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
```

Comment:
UTF-8 is the default encoding, but could you add it to all the XML files so we keep it standard accross all our rules ? <?xml version="1.0" encoding="UTF-8"?>

Reply:
Updated as suggested